js/clinicalTimeline.js

```diff
@@ -782,27 +782,38 @@ window.clinicalTimeline = (function(){
       var dayFormat = [];
       var m;
       var d;
-      if (time.y !== 0) {
-        dayFormat = dayFormat.concat(time.y+"y");
-      }
-      if (time.m !== 0) {
-        if (time.y !== 0) {
-          m = Math.abs(time.m);
-        } else {
-          m = time.m;
-        }
-        dayFormat = dayFormat.concat(m+"m");
-      }
-      if (time.d !== 0) {
-        if (time.y !== 0 || time.m !== 0) {
-          d = Math.abs(time.d);
-        } else {
-          d = time.d;
-        }
-        dayFormat = dayFormat.concat(d+"d");
-      }
+      var daysPerYear = 365;
+      var daysPerMonth = 30;
+
       if (time.y === 0 && time.m === 0 && time.d === 0) {
         dayFormat = [0];
+      } else {
+        if (getZoomLevel(beginning,ending,width) == "days" || getZoomLevel(beginning,ending,width) == "10days" || getZoomLevel(beginning,ending,width) == "3days") {
```

Comment:
I prefer to use triple `===` for comparison. I guess in this case it doesn't matter much but using double `==` often leads to more tricky corner cases so I tend to always use `===`. See also http://stackoverflow.com/questions/359494